src/backend/utils.py

```diff
@@ -50,3 +50,60 @@ def plot_scree_test(eigen_values):
                      markerscale=0.4)
     leg.get_frame().set_alpha(0.4)
     plt.show()
+
+def get_most_m_similar_images(data_with_images, query_image_feature_vector, Vt, m):
+    """
+    Author: Vibhu Varshney
+    This funcion computes the similarity score between the query image vector and the images in the database
+    :param data_with_images_: This is a dict/map with image name list and the data matrix
+    :param query_image_feature_vector : Query Image feature vector after applying the feature extraction model
+    :param Vt: This is the latent-vector by original feature matrix generated from either model
+    :param m: Number of similar images to be returned
+    :return: dictionary of m most similar images as keys and their scores as value
+    """
+    db_data_matrix = data_with_images.get('data_matrix')
+    imageNames = data_with_images.get('images')
+    database_images_latent_vectors = np.dot(db_data_matrix, np.transpose(Vt))
+    query_image_latent_vector = np.dot(np.array(query_image_feature_vector),Vt.T)
+    return get_top_m_tuples_by_similarity_score(database_images_latent_vectors, query_image_latent_vector, imageNames, m+1) #+1 because the db contains the query image also
```

Comment:
** Just a suggestion **

If possible use IDE (Pycharm, Visual Code) for coding and follow python flake8 coding style. 

For ex:
- Break line into multiple if length goes beyond 120 or 240 chars. It will be easy to read for you and reviewer.

Reply:
Done sir